### DISCUSSION TITLE
Webhook for PriceList/update

### DISCUSSION THREAD

**Discussion by rcigni** [Shopify/storefront-api-feedback#79] [Ideas & Requests]
We build beautiful headless commerce with Shopify and we want to use Markets as soon as possible but it's really important to have webhooks to interact with PiceList (especially after a priceListFixedPricesAdd) so that we can keep in sync external systems that we use to serve the site like Algolia for the search and PLP. 

Without it, we have to distribute the change on two different endpoints, making the implementation less reliable and shopify-centric.

**Comment by stevengibbsonline** [comment]
@rcigni Did you find a solution or any other event topics firing off for this?
